.bin/esy

```diff
@@ -96,19 +96,24 @@ builtIn() {
 	node $SCRIPTDIR/esy.js $@
 }
 
-if [ "$1" == "build" ] || [ "$1" == "shell" ] || [ "$1" == "clean" ]; then
+if [ "$1" == "build" ] || [ "$1" == "build-shell" ] || [ "$1" == "clean" ]; then
   builtInEject
   make -j -s -f "$EJECT_PATH/Makefile" "$1"
 
 elif [ "$1" == "build-eject" ]; then
   builtInEject
 
+elif [ "$1" == "shell" ]; then
+  ensureEnvEjected
+  source "$ENV_PATH"
+  $SHELL
```

Comment:
I'd really like to customize `PS1` but not sure how to do that... suggestions welcome.